### PR TITLE
Added moving car animation

### LINKED ISSUE #629: Moving car animation effect
I have created a moving car animation effect. I want to add it. Please assign it to me.

### PR DESCRIPTION
**Pull request by Areebah03**
*Fixed #629

### ✅ Check List (Check all the applicable boxes) 

- [x] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [x] This PR does not contain plagiarized content.
- [x] My Addition/Changes works properly and matches the overall repo pattern.
- [x] The title of my pull request is a short description of the requested changes.

### 📄 Note to reviewers

It is an animation of a moving car, with music feature build using HTML, CSS, JavaScript

### 📷 Screenshots

movingcar

### CODE REVIEW

**Comment by Areebah03** [comment]
@Astrodevil  I have made a new PR for moving car animation, Please view it.

File: cards.json [hunk 1]
```diff
@@ -709,5 +709,10 @@
         "name": "YouTube Clone",
         "image": "assets/Images/YouTubeClone/YouTubeClone.png",
         "link": "Public/YouTube_Clone/index.html"
+    },
+    {
+        "name": "Moving Car",
```
Comment (Astrodevil) [inline cards.json]: Update this Alphabetically

**Comment by Astrodevil** [comment]
Still sound is not working

**Comment by Areebah03** [comment]
@Astrodevil There is some problem with the sound :( so I have removed it.

**Pull request merged**